src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSuplier {
+    public static final String DEFAULT_COLOR = "White";
+    public static final int DEFAULT_RADIUS = 10;
+    private static ColorSuplier colorSuplier = new ColorSuplier();
+    private static Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(5)) {
+            case 0: return createCircle();
+            case 1: return createRectangle();
+            case 2: return createSquare();
+            case 3: return createTrapezoid();
+            case 4: return createRightTriangle();
+            default: return getDefaultFigure();
+        }
+    }
+
+    public Figure createRectangle() {
+        return new Rectangle(colorSuplier.getRandomColor(),random.nextInt(),random.nextInt());
```

Comment:
```suggestion
        return new Rectangle(colorSuplier.getRandomColor(), random.nextDouble(), random.nextDouble());
```
Let’s use white spaces (check it everywhere). Also, let’s use random.nextDouble() everywhere